snap/local/runtime-helpers/bin/startup-env-var.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+EDGEX_STARTUP_DURATION=$(snapctl get startup-duration)
+export EDGEX_STARTUP_DURATION
```

Comment:
I think you still want to check that the variables aren't empty( via `if [ -n $VAR` ] ) before setting it, as I'm not sure what the behavior is when the environment is defined but has no value.